breadcord/helpers.py

```diff
@@ -24,7 +24,7 @@
 
 
 async def administrator_check(interaction: discord.Interaction) -> bool:
-    """A discord.py check to ensure that the interaction user is an administrator.
+    """discord.py check to ensure that the interaction user is an administrator.
```

Comment:
This isn't imperative mood as far as I know, so I'm not sure why this change was made - I preferred it with the `A` left in personally. Any reason for the removal?

```suggestion
    """A discord.py check to ensure that the interaction user is an administrator.
```